crates/services/tx_status_manager/src/manager.rs

```diff
@@ -169,7 +180,11 @@ impl TxStatusManager {
 
         self.tx_status_change
             .update_sender
-            .send(TxUpdate::new(tx_id, tx_status.into()));
+            .send(TxUpdate::new(tx_id, tx_status.clone().into()));
+
+        if let Err(e) = self.tx_status_change_sender.send((tx_id, tx_status)) {
+            tracing::error!(%e, "failed to send tx status update");
```

Comment:
```suggestion
            tracing::warn!(%e, "failed to send tx status update");
```

I'm even not sure do we need to print it=D But if you want, let's at least use warning here. Because it is not an error=)

Reply:
Changed